assets/js/googlesitekit/widgets/components/Widget.js

```diff
@@ -37,6 +53,8 @@ Widget.defaultProps = {
 Widget.propTypes = {
 	children: node,
 	slug: string.isRequired,
+	header: func,
+	footer: func,
```

Comment:
I'm not sure, is there a general preference in React? We currently require a component function in #1623, so I went with the same approach here. I'd rather decide on one consistently rather than supporting either.

Reply:
I went with `elementType` since that's what we pass in #1623.

Reply:
Sounds good, I think that's what we want anyways. An `elementType` is like a reference to a "component definition". So this could be an arrow function that returns something else (i.e. an anonymous wrapper component), or a reference to the component function itself. The main difference is that an `elementType` can be passed props when it's used, because we're kind of "instantiating" it. You can think of an `element` (i.e. the result of `createElement`) as an instance which is already instantiated, so we can't pass any props to it.